src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,79 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Returns the collection that holds the tasks. Should only be used to
+     * iterate over the list.
+     *
+     * @return Collection of Tasks.
+     */
+    public ArrayList<Task> getList() {
+        return tasks;
+    }
+
+    /**
+     * Adds a task to the task list.
+     *
+     * @param task
+     * @return true.
+     */
+    public boolean add(Task task) {
```

Comment:
Maybe this can be renamed as users might not be able to tell that it is a boolean on first look, maybe taskIsAdded?